magprime/models.py

```diff
@@ -15,7 +15,7 @@ def invalid_notification(self):
     @cost_property
     def child_discount(self):
         if 'val' in self.age_group_conf and self.age_group_conf['val'] == c.UNDER_13:
-            return (c.BADGE_PRICE / 2) * -1
+            return math.ceil(c.BADGE_PRICE / 2) * -1
```

Comment:
Minor syntax note for future reference: you can just prefix an expression with ``-`` to make it negative, i.e.

```python
return -math.ceil(c.BADGE_PRICE / 2)
```

This avoids needing to multiply by -1.  Though maybe you prefer the latter stylistically.  I have no strong preference on this myself.